optimum/habana/accelerate/utils/transformer_engine.py

```diff
@@ -75,6 +77,17 @@ def _convert_model(model, to_transformer_engine=True, _convert_linear=True):
                 new_module.bias.copy_(module.bias)
 
             setattr(model, name, new_module)
+        elif isinstance(module, ModuleFusedSDPA) and module.flash_attention_fp8 and to_transformer_engine:
+            from habana_frameworks.torch.hpex.experimental.transformer_engine import (
+                FusedAttention as te_FusedAttention,
+            )
+
+            module._hpu_kernel_fsdpa = te_FusedAttention(
```

Comment:
```suggestion
                FusedAttention as FusedAttentionTE,
            )

            module._hpu_kernel_fsdpa = FusedAttentionTE(
```

Please be consistent with other code. Below, at `modeling_llama.py` are using `FusedAttentionTE`